src/App.js

```diff
@@ -44,9 +38,16 @@ const notifyUser = announcement => {
 }
 
 const PageRoute = ({ path, children }) => {
+  const [livesiteDoc, setLivesiteDoc] = useState(false)
```

Comment:
Nit: I would set this to `null`, then use an optional chaining operator on line 50 to prevent "cannot read property of null" errors - lmk though! Something about having the default state for an object be `false` seems off to me, but it doesn't seem to have any functional impact either way

Reply:
good catch :)